src/main/java/at/hannibal2/skyhanni/features/event/diana/ArrowGuessBurrow.kt

```diff
@@ -0,0 +1,223 @@
+package at.hannibal2.skyhanni.features.event.diana
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.ReceiveParticleEvent
+import at.hannibal2.skyhanni.events.diana.BurrowGuessEvent
+import at.hannibal2.skyhanni.features.event.diana.GriffinBurrowHelper.allowedBlocksAboveGround
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.BlockUtils.getBlockAt
+import at.hannibal2.skyhanni.utils.BlockUtils.isInLoadedChunk
+import at.hannibal2.skyhanni.utils.LocationUtils.isInside
+import at.hannibal2.skyhanni.utils.LorenzVec
+import at.hannibal2.skyhanni.utils.RaycastUtils
+import at.hannibal2.skyhanni.utils.collection.TimeLimitedSet
+import net.minecraft.init.Blocks
+import net.minecraft.util.EnumParticleTypes
+import kotlin.time.Duration.Companion.minutes
+
+@SkyHanniModule
+object ArrowGuessBurrow {
+    private val config get() = SkyHanniMod.feature.event.diana
+
+    private val points: HashSet<LorenzVec> = HashSet()
```

Comment:
i dont think this needs to specifically be a hash set, you can just use kotlin's mutable set (which is a linked hash set, and its what we mostly use in the codebase)
```suggestion
    private val points: MutableSet<LorenzVec> = mutableSetOf()
```